install/networks-default.yml

```diff
@@ -0,0 +1,110 @@
+networks:
+  - name: mainnet
+    label: Ethereum Mainnet
+    description: This is the real Ethereum main network, using real ETH and real RPL to make real validators.
+    txWatchUrl: "https://etherscan.io/tx"
+    stakeUrl: "https://stake.rocketpool.net"
+    chainID: 1
+    flashbotsProtectUrl: "https://rpc.flashbots.net/"
+    snapshotApiDomain: "hub.snapshot.org"
+    addresses:
+      storage: "0x1d8f8f00cfa6758d7bE78336684788Fb0ee0Fa46"
+      rplToken: "0xD33526068D116cE69F19A9ee46F0bd304F21A51f"
+      rplFaucet: ""
```

Comment:
here, and elsewhere, we should be able to leave these addresses blank.

Perhaps `mainnet` should have them all, as a form of documentation, even the empty ones.

However, devnet/holesky it becomes mostly noise